lib/cisco_node_utils/cisco_cmn_utils.rb

```diff
@@ -96,6 +96,13 @@ def self.generic_prop_set(klass, rlbname, props)
   # General utility class
   class Utils
     require 'ipaddr'
+
+    # Helper utility to check for older Nexus I2 images
+    def self.nexus_i2_image
+      require_relative 'platform'
+      true if Platform.image_version[/7.0.3.I2.*/]
```

Comment:
Nope. Originally I was returning the version info to the caller instead of true so I wanted to keep it. I'll remove it.